addons/point_of_sale/static/tests/tours/PosComboTour.js

```diff
@@ -0,0 +1,74 @@
+/** @odoo-module */
+
+import { ProductScreen } from "@point_of_sale/../tests/tours/helpers/ProductScreenTourMethods";
+import { PaymentScreen } from "@point_of_sale/../tests/tours/helpers/PaymentScreenTourMethods";
+import { registry } from "@web/core/registry";
+import { Product } from "@point_of_sale/app/store/models";
+const label = (productName) =>
+    `label.combo-line:has(article.product .product-name:contains("${productName}"))`;
+const isNot = (trigger) => `body:not(:has(${trigger}))`;
+const isComboSelectedTrigger = (productName) => `${label(productName)} ~ input:checked`;
+
+const isComboSelected = (productName) => ({
+    content: `Check that ${productName} is selected`,
+    trigger: isComboSelectedTrigger(productName),
+    isCheck: true,
+});
+const isNotComboSelected = (productName) => ({
+    content: `Check that ${productName} is not selected`,
+    trigger: isNot(isComboSelectedTrigger(productName)),
+    isCheck: true,
+});
+
+registry.category("web_tour.tours").add("PosComboTour", {
+    test: true,
+    url: "/pos/ui",
+    steps: () => [
+        ProductScreen.do.confirmOpeningPopup(),
+
+        ProductScreen.exec.addOrderline("Office Combo"),
+        {
+            content: "check that the combo popup appeared",
+            trigger: ".popup.combo-configurator-popup",
+            isCheck: true,
+        },
+        {
+            content: "check that amount is not displayed if zero",
+            trigger: `article.product .product-content:not(:has(.price-tag:contains("0")))`,
+        },
+        {
+            content: "check that amount is properly displayed when it is not 0",
+            trigger: `article.product .product-content .product-name:contains("Combo Product 3") ~.price-tag:contains("$2.00")))`,
+        },
+        ProductScreen.do.clickDisplayedProduct("Combo Product 1"),
+        {
+            content: "try to click `confirm` without having made all the selections",
+            trigger: "footer button.confirm[disabled]",
+            isCheck: true,
+        },
+        ProductScreen.do.clickDisplayedProduct("Combo Product 5"),
+
+        ProductScreen.do.clickDisplayedProduct("Combo Product 7"),
+        isComboSelected("Combo Product 7"),
+        ProductScreen.do.clickDisplayedProduct("Combo Product 8"),
+        isComboSelected("Combo Product 8"),
+        isNotComboSelected("Combo Product 7"),
+        {
+            content: "check that the total amount is properly displayed",
+            trigger: `combo-configurator-popup .total-amount:contains("$2402.00")`,
+            isCheck: true,
+        },
+        {
+            content: "Click `confirm`",
+            trigger: "footer button.confirm",
+        },
+        ProductScreen.do.clickOrderline("Office Combo"),
+        ProductScreen.do.clickOrderline(""),
+
+        // check that all the products are added to the order
+
+
+
```

Comment:
could be great to check here the total price, with possible extra combo_price. 

+ Check vat. (see my main comment)

Reply:
I agree, i'll certainly do that. Thanks!